.github/workflows/test.yml

```diff
@@ -77,7 +77,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.19.x]
+        go-version: [1.20.x]
```

Comment:
This looks like something went wrong on a rebase

Reply:
Nop, it is expected. I tried to apply it to see if we would have any issues with 1.20, but I will remove soon.